workflow/controller/operator.go

```diff
@@ -2121,9 +2152,8 @@ func (woc *wfOperationCtx) markNodeError(nodeName string, err error) *wfv1.NodeS
 
 // markNodePending is a convenience method to mark a node and set the message from the error
 func (woc *wfOperationCtx) markNodePending(nodeName string, err error) *wfv1.NodeStatus {
-	woc.log.Infof("Mark node %s as Pending, due to: %+v", nodeName, err)
-	node := woc.wf.GetNodeByName(nodeName)
-	return woc.markNodePhase(nodeName, wfv1.NodePending, fmt.Sprintf("Pending %s", time.Since(node.StartedAt.Time)))
+	woc.log.Infof("Mark node %s as Pending, due to: %v", nodeName, err)
+	return woc.markNodePhase(nodeName, wfv1.NodePending, err.Error())
```

Comment:
adding the timestamp to the pending was causing the workflow to be updated when it had not changed and therefore excessive reconciliation